src/app/components/merge/ExtendedContactSummary.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Block, Icon } from 'react-components';
+
+import { formatAdr } from '../../helpers/property';
+import { addPref, sortByPref } from '../../helpers/properties';
+
+const getFirstValue = (properties, field) => {
+    const { value } = properties.find(({ field: f }) => f === field) || {};
+    return Array.isArray(value) ? value.join(', ') : value;
+};
+
+const getAllValues = (properties, field) => {
+    return addPref(properties)
```

Comment:
Here you can do `.filter`, '.sort`, `.map`.

Reply:
Yep, I was on a rampage with `reduce` :sweat_smile: